unit_tests/test_argus.py

```diff
@@ -0,0 +1,26 @@
+import os
+import logging
+from unittest import mock
+
+import pytest
+
+from sdcm.argus_test_run import ArgusTestRun
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.integration
+@pytest.mark.need_network
+@mock.patch.dict(os.environ, {
+    "JOB_NAME": "test_connectivity",
+})
+@pytest.mark.sct_config(files='test-cases/PR-provision-test-docker.yaml')
+def test_connectivity(params):
+    argus_test_run = ArgusTestRun.from_sct_config(test_id='test_connectivity', sct_config=params)
```

Comment:
`test_id` should be a UUID, and you can't really re-use a single one as that will lead to multiple rows with same test id (since id is not the primary key)

Reply:
I don't really use it. i.e. I'm not calling save.

I just want to make sure it's connecting, not fill the DB with garbage.

Reply:
other suggestions on how to achieve that goal are welcomed

Reply:
also now the test is failing:
```
09:32:59          if cls._config is None:
09:32:59  >           cls._config = ArgusConfig(**KeyStore().get_argusdb_credentials(), keyspace_name="argus")
09:32:59  E           TypeError: Config.__init__() got an unexpected keyword argument 'address_mapping'
```

since the `ArgusConfig` is a bit unfriendly for extensions, using it like this.
cause of this I'm force to have multiple versions of the configuration (since this configuration is shared across SCT branches)

Reply:
> I don't really use it. i.e. I'm not calling save.
> 
> I just want to make sure it's connecting, not fill the DB with garbage.

Oh, right, then ignore that comment. Yeah that should be enough to just test the connectivity as connection is established as soon as the ArgusDatabase class is instantiated.